### ISSUE TITLE
Transparency/Opcaity in the Water Mark Options

### ISSUE THREAD

**Issue by drsimplegraffiti** [ranjandsingh/sharp-multer#1]
```
watermarkOptions:  {
        input:  "./images/logo.png",
        location:  "top-right",
        },
```

Is there any way to reduce the opacity/transparency in the water mark options? If no, can it be added

**Comment by ranjandsingh** [comment]
Currently opacity/transparency is not supported yet but can be done needs little experimentation

**Comment by ranjandsingh** [comment]
just released version 0.2.2 with this feature have fun @drsimplegraffiti
